Gemfile

```diff
@@ -8,6 +8,7 @@ group :development do
   gem "rspec"
   gem "rdoc"
   gem "rake"
+  gem "rubocop", require: false
```

Comment:
Oracle enhanced adapter does not repository manage the Gemfile.lock. So I think it's better to specify the version to Gemfile.

```diff
-gem "rubocop", require: false
+gem "rubocop", "~> 0.63", require: false
```

We update it with the following channel of Code Climate when bump the version of RuboCop.
https://github.com/rsim/oracle-enhanced/blob/v5.2.5/.codeclimate.yml#L4

BTW, it seems necessary to update channel from `channel: rubocop-0-51` to `channel: rubocop-0-63` in .codeclimate.yml of this repository.
https://github.com/rails/rails/blob/4ad4de50ce164f7b62661408eceb4259dfccdf66/.codeclimate.yml#L26

Reply:
Thanks for the review. 

I think you suggest to use the same (minor) version of RuboCop between `Gemfile` and `.codeclimate.yml`. 

Do you mean to use ``gem "rubocop", "~> 0.63.0", require: false` adding teeny version like `0.63.0` not `0.63`

Let me know if I misunderstand.

* `gem "rubocop", "~> 0.63", require: false` installs RuboCop 0.65.0.

```
$ bundle info rubocop
... snip ..
  * rubocop (0.65.0)
	Summary: Automatic Ruby code style checking tool.
	Homepage: https://github.com/rubocop-hq/rubocop
	Path: /home/yahonda/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/rubocop-0.65.0
$
```


 * `gem "rubocop", "~> 0.63.0", require: false` installs RuboCop 0.63.1
```
$ bundle info rubocop
... snip ...
  * rubocop (0.65.0)
	Summary: Automatic Ruby code style checking tool.
	Homepage: https://github.com/rubocop-hq/rubocop
	Path: /home/yahonda/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/rubocop-0.65.0
```

Reply:
Usually I maintain .codeclimate.yml of master branch. https://github.com/rsim/oracle-enhanced/blob/94be7a3404be2d42be703492ef7c4ba4719ca2eb/.codeclimate.yml#L26
I think we are running `rubocop-0-63`. Not updating .codeclimate.yml of release52 branch, though.

Reply:
> Do you mean to use `gem "rubocop", "~> 0.63.0", require: false` adding teeny version like `0.63.0` not `0.63`?

Ah, I made a mistake, it's actually mean `gem "rubocop", "~> 0.63.0", require: false`.

> Usually I maintain .codeclimate.yml of master branch.

I'm sorry. I mistook what I saw 💦

Reply:
Nevermind. Updated the pull request.